client/signup/steps/woocommerce-install/hooks/use-woop-handling/index.ts

```diff
@@ -17,8 +17,8 @@ import hasAvailableSiteFeature from 'calypso/state/selectors/has-available-site-
 import { getSiteDomain } from 'calypso/state/sites/selectors';
 
 const TRANSFERRING_NOT_BLOCKERS = [
-	eligibilityHoldsConstants.NO_BUSINESS_PLAN, // let's redirect to checkout page
-	// eligibilityHoldsConstants.TRANSFER_ALREADY_EXISTS, // ToDo: let's handle it in transferring endpoint.
```

Comment:
This was blocking installation on the landing page

Reply:
👍